frontend/src/app/main/data/workspace.cljs

```diff
@@ -273,6 +273,16 @@
         (rx/of (dws/select-shapes frames-id)
                dwz/zoom-to-selected-shape)))))
 
+(defn- select-frame-tool
+  [{:keys [page-id file-id]}]
```

Comment:
I think we don't need here a map for passing two static positional params. Avoid unnecesary allocation with a function with two arguments

```clojure
(defn- select-frame-tool
  [file-id page-id]
       ...
```